src/components/TimelineNavPanel/TimelineNavPanel.tsx

```diff
@@ -57,6 +61,7 @@ export const TimelineNavPanel = ({
   className,
   completed,
   id,
+  right,
```

Comment:
Where is this being used?

Reply:
Looks like all of `<TimelineNavPanel>`'s props get passed to `<TimelineNav>`. In `<TimelineNav>` the panels, props and children get collected by timelineNavItems(), which gets iterated over by a .map() method inside the return; the props then get applied to each <li>.

Reply:
Could you add a quick comment, here and on the other one?